### DISCUSSION TITLE
Anyway to modify the query for the resources included on the Overlook Plugin?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [awcodes/overlook#29] [Ideas]
I want to modify the query for some of the resources being included on the Overlook Plugin. For example, I want to show a list of Users that are pending to be approved. I am using the `UserResource` but I want to add a filter by modifying the query only for the overlook widget. Please advice how I can do this.

**Comment by awcodes** [comment]
Currently not a way to do that. Could be a good PR though. But right now it's just reading directly off the resource.

**Reply by abishekrsrikaanth** [reply]
Created a PR https://github.com/awcodes/overlook/pull/30. Can you review and let me know what you think.
